dask/dataframe/tests/test_groupby.py

```diff
@@ -3172,6 +3172,12 @@ def test_groupby_numeric_only_true(func):
 
 @pytest.mark.parametrize("func", ["cov", "corr"])
 def test_groupby_numeric_only_false_cov_corr(func):
+
+    if PANDAS_GE_300 and func == "cov":
```

Comment:
Thanks @TomAugspurger! 

Thoughts on changing this to only add the `"timedelta"` column when it's supported? That way we still test `cov` works for the other dtypes